app/views/groups/show.html.erb

```diff
@@ -1,19 +1,26 @@
 <p id="notice"><%= notice %></p>
-
+<p>
+  <strong>Administrator:</strong>
+  <%= @group.owner.email %>
+</p>
 <p>
   <strong>Group Name:</strong>
   <%= @group.name %>
 </p>
-
 <p>
   <strong>Group Description:</strong>
   <%= @group.description %>
 </p>
+<p>
+  <strong>Group Amount:</strong>
+  <%=number_to_currency( @group.amount,locale: :en) %>
```

Comment:
Its a small one, but just check the spaces in the `( )` at the beginning

Reply:
I done this. Now there isn't any space at the beginning of ( ) . Thanks for your CR